src/targets/gpu/mlir.cpp

```diff
@@ -487,6 +488,17 @@ struct mlir_program
             ops.add_attribute_value(get_operator_value(ins->get_operator()));
             if(ins->name() != "@return")
                 ops.add_results({get_shape(ins)});
+            if(ins->name() == "convolution")
+            {
+                pp =
+                    problem_params{ins->get_operator(), to_shapes(ins->inputs()), ins->get_shape()};
+                std::string tuned = get_tune_params();
+                if(!tuned.empty())
+                    ops.add_attributes({{"perf_config", tuned}});
+                // check if HW supports xdlops
+                if(target_name.compare("gfx908") == 0 || target_name.compare("gfx90a") == 0)
```

Comment:
I think you should make an unordered set of these architectures so that we can add more if relevant in the future:
```
const std::unordered_set<std::string>& get_xdlops_archs()
    {
        static std::unordered_set<std::string> supported_archs{"gfx908", "gfx90a"};
        return supported_archs;
    }
```
Then you can do:
```
if(contains(get_xdlops_archs(), device_name))
```